extension/mysql_to_doris/bin/e_mysql_to_doris.sh

```diff
@@ -31,8 +31,8 @@ rm -f $path
 #get create table sql for mysql
 for table in $(cat ../conf/mysql_tables |grep -v '#' | awk -F '\n' '{print $1}')
         do
-        d_d=`echo $table` |awk -F '.' '{print $1}'
-        d_t=`echo $table` |awk -F '.' '{print $2}'
+        d_d=`echo $table |awk -F '.' '{print $1}'`
+        d_t=`echo $table |awk -F '.' '{print $2}'`
```

Comment:
It is better to use this syntax.
```suggestion
        d_d=$(echo $table |awk -F '.' '{print $1}')
        d_t=$(echo $table |awk -F '.' '{print $2}')
```

Reply:
BTW, some checks failed. Please rebase the code to trigger them again.

Reply:
OK,That's right